htdocs/core/lib/functions.lib.php

```diff
@@ -10422,7 +10422,7 @@ function dolGetButtonAction($label, $html = '', $actionType = 'default', $url =
 	global $hookmanager, $action, $object, $langs;
 
 	//var_dump($params);
-	if ($params['isDropdown'])
+	if (isset($params['isDropdown']))
```

Comment:
Should be !empty for the case value is 0

Reply:
@eldy Ok for me it's done